doc/theory/glossary.tex

```diff
@@ -213,10 +212,19 @@
 	\begin{equation*}
 	||\lambda||_p = \left( \sum_{i \in \mathbb N} ||\lambda_i||^p_p \right)^{1/p}
 	\end{equation*}
-	whenever the right hand side exists and is finite. 
+	whenever the right hand side exists and is finite.
```

Comment:
There is also an extra space I missed in the subsection name

Reply:
Not sure I understand what you mean, sorry.